src/field/field255.rs

```diff
@@ -289,6 +289,24 @@ impl Default for Field255 {
     }
 }
 
+impl TryFrom<Field255> for u64 {
+    type Error = FieldError;
+
+    fn try_from(elem: Field255) -> Result<u64, FieldError> {
+        let mut le_bytes = [0; 32];
+        fiat_25519_to_bytes(&mut le_bytes, &elem.0);
+        let mut is_zero = Choice::from(1);
+        for byte in &le_bytes[8..] {
```

Comment:
nit: to make it more clear why we're checking that the last 24 bytes are 0, `use std::mem::size_of` and:
```suggestion
        for byte in &le_bytes[size_of::<u64>()..] {
```